main/test/flix/fuzzers/FuzzDeleteLines.scala

```diff
@@ -22,21 +22,16 @@ import org.scalatest.funsuite.AnyFunSuite
 
 import java.nio.file.{Files, Paths}
 
-class FuzzLines extends AnyFunSuite with TestUtils {
+class FuzzDeleteLines extends AnyFunSuite with TestUtils {
 
-  test("simple-card-game") {
-    val input = Files.lines(Paths.get("examples/simple-card-game.flix"))
-    compileAllLinesLessOne(input)
-  }
-
-  test("the-ast-typing-problem-with-polymorphic-records") {
-    val input = Files.lines(Paths.get("examples/the-ast-typing-problem-with-polymorphic-records.flix"))
-    compileAllLinesLessOne(input)
-  }
+  private val testFiles = List(
+    "simple-card-game" -> Files.lines(Paths.get("examples/simple-card-game.flix")),
+    "the-ast-typing-problem-with-polymorphic-records" -> Files.lines(Paths.get("examples/the-ast-typing-problem-with-polymorphic-records.flix")),
+    "using-channels-and-select" -> Files.lines(Paths.get("examples/using-channels-and-select.flix")),
+  )
 
-  test("using-channels-and-select") {
-    val input = Files.lines(Paths.get("examples/using-channels-and-select.flix"))
-    compileAllLinesLessOne(input)
+  testFiles.foreach {
+    case (name, input) => test(s"$name-delete-lines")(compileAllLinesLessOne(name, input))
```

Comment:
We try hard to avoid programmatic construction of test cases because it sometimes breaks in IDEA etc.

So, I would really prefer to go back to the old style.